ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -199,6 +199,362 @@ def _grpc_channel_pool(self) -> GrpcChannelPool:
         return self._session_management_client._grpc_channel_pool
 
 
+class MultiplexerSessionHandler(_BaseSessionContainer):
```

Comment:
I'm not 100% sold on the word "handler" because it's commonly associated with event callbacks.

"accessor" or "container" would make more sense to me.

@dixonjoel What do you think?

Reply:
I went ahead with `MultiplexerSessionContainer` as it sounds better. Please let me know if that's fine.